src/utils/getTeams.test.ts

```diff
@@ -4,16 +4,16 @@ import { getTeams } from './getTeams';
 
 // Check tests/product-owners.yml for data used for tests here
 describe('getTeams', function () {
-  it('should return empty array if no teams were found', () => {
-    expect(getTeams('repo-does-not-exist', 'getsentry')).toEqual([]);
+  it('should return team-ospo if no teams were found', () => {
```

Comment:
Wonder if its not worth making a new team, `team-unknown` or similar, specifically for handling this case? Otherwise our channel will potentially be flooded, mixing things that are a: actually our team's tasks with b: things that are not our team's responsibility, but have fallen through to us due to configuration issues.

Reply:
Technically, we are in charge of the process of making sure issues are appropriately routed to the teams that are in charge of these issues. I was thinking the default behavior would be to assign those issues to us, not to close, but to make sure they get to the appropriate teams that should own the issue. 

That being said, it made sense to me to assign `team-ospo` here since that means that we should figure out the correct team to route an issue to.